src/test/java/com/practise/tdd/WordCountShould.java

```diff
@@ -0,0 +1,19 @@
+package com.practise.tdd;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class WordCountShould {
+
+	WordCount wordCount;
+	
+	@Before
+	public void setup() {
+		wordCount = new WordCount();
+	}
+	
+	@Test
+	public void count_unique_words_in_single_word_string() {
```

Comment:
As per course material, 'Not compiling' is a failure. So I already had a Red, which needed fixing, as first step.